plugin_gems.rb

```diff
@@ -13,7 +13,7 @@
 download "webhdfs", "0.8.0"
 download "fluent-plugin-webhdfs", "1.2.3"
 download "fluent-plugin-rewrite-tag-filter", "2.2.0"
-download "fluent-plugin-google-cloud", "0.8.7"
+download "fluent-plugin-google-cloud", "0.9.0"
```

Comment:
Sorry about not noticing this before — this version must be an already released version; it gets bumped as part of the actual release. We need to back this part of the change out because it breaks the build, since it's unable to download the version. See #272.